File: android-app/app/src/main/java/ru/mxk/userslist/model/Person.kt
package ru.mxk.userslist.model

import com.google.gson.annotations.SerializedName
import java.util.UUID

data class Person(
    val id: UUID,
    val name: String,
    @SerializedName("company")
    val companyName: String,
    @SerializedName("avatar")
    val photo: String,
    val liked: Boolean,
    val fired: Boolean,
    val active: Boolean,
    val details: String
)

File: ios-app/PersonListApplication/model/Person.swift
import Foundation


struct Person: Identifiable {
    let id: String
    let name: String
    let companyName: String
    let photo: String
    let liked: Bool
    let fired: Bool
    let active: Bool
    let details: String
}

extension Person: Decodable, Encodable {
    enum CodingKeys: String, CodingKey {
        case id, name, liked, fired, active, details
        case companyName = "company"
        case photo = "avatar"
    }
}

extension Person: CustomStringConvertible {
    var description: String {
        return "Person(id: \(id), name: '\(name)')"
    }
}
